apps/disease_control/migrations/0004_diseasepage.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.1.6 on 2021-03-01 12:01
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+import wagtail.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("wagtailcore", "0060_fix_workflow_unique_constraint"),
+        ("disease_control", "0003_diseasesandconditionspage"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="DiseasePage",
+            fields=[
+                (
+                    "page_ptr",
+                    models.OneToOneField(
+                        auto_created=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        parent_link=True,
+                        primary_key=True,
+                        serialize=False,
+                        to="wagtailcore.page",
+                    ),
+                ),
+                ("description", wagtail.core.fields.RichTextField(blank=True)),
+                (
+                    "is_emergent",
+                    models.BooleanField(
+                        default=False, help_text="Is this an emergent disease?"
```

Comment:
Putting my doctor hat on, for these three fields I would prefer help text of:
* is_emergent: Should this disease be tagged emergent?
* emergent_begin_date: Choose the date this became an emergent concern
* emergend_end_date: Choose the date this stopped being an emergent concern. Leave blank if it is still an emergent concern.

Reply:
Good thinking